ANSWERS.md

```diff
@@ -1,6 +1,15 @@
 ### Questions:
 1. What is the DOM?
+    The DOM is the Document Object Model. Basicly it is a live representation of the rendered HTML. It gives us a way to manipulate the content, stucture and style of the page using JavaScript. 
+
 2. What is an event?
+    An event is anything that happens in the browser. A few examples of this would be mouse movements, clicks, resize window and scroll.
+
 3. What is an event listener?
+    An event listener is a peice of code that is assigned the job of looking for a specific event to occer on a specific object or on the website in general. It allows us to manipulate the page only after a specific event occurs, like a click. 
+
 4. Why would we convert a NodeList into an Array?
+    With the NodeList we can only use the built in function of .forEach. In order to get access to all of the array methods we must turn in into a real array. This gives us access to things like .map().
```

Comment:
Just to make things clear, there are more methods native to the NodeList data type.